web/packages/shared/components/UnifiedResources/FilterPanel.tsx

```diff
@@ -531,29 +520,15 @@ const IncludedResourcesSelector = ({
     setAnchorEl(null);
   };
 
-  function applyFilter(e: FormEvent<HTMLFormElement>) {
-    e.preventDefault();
-    handleClose();
-
-    const formData = new FormData(e.currentTarget);
-    const availabilityOptionsForm = formData.getAll('availabilityOptions');
-
-    if (availabilityOptionsForm.length === 0) {
-      onChange('none');
+  function handleToggle() {
+    if (
+      availabilityFilter.mode === 'requestable' ||
+      availabilityFilter.mode === 'all'
```

Comment:
It would be nice if these were enums

Reply:
(it's a union at least)
But it seems that we should get rid of the now unused `none` option.

Reply:
we'll have to keep none around for a little bit to ensure users who had it selected will still work correctly